engine/src/flutter/impeller/entity/entity_pass_clip_stack.cc

```diff
@@ -83,6 +83,8 @@ EntityPassClipStack::ClipStateResult EntityPassClipStack::RecordRestore(
   if (subpass_state.clip_coverage.back().coverage.has_value()) {
     FML_DCHECK(next_replay_index_ <=
                subpass_state.rendered_clip_entities.size());
+    // This code is slightly wrong and should be popping more than one clip
+    // entry.
```

Comment:
File an issue and add a TODO with the link.

Reply:
Done